MolecularNodes/ui.py

```diff
@@ -86,12 +87,18 @@ def MOL_PT_panel_rcsb(layout_function, ):
     col_main.scale_y = 1.0
     col_main.alignment = 'Expand'.upper()
     col_main.label(text = "Download from PDB")
+    row_cache = col_main.row()
+    row_cache.prop(
+        bpy.context.scene,
+        'mol_cache_dir',
+        text = 'Cache dir',
+        icon_value = 0,
```

Comment:
I've got it a couple of places in my code, but it turns out we don't need `emboss = True` and `icon_value = 0` as these are the defaults anyway, so you can trim these if you want.